dash/include/dash/algorithm/Accumulate.h

```diff
@@ -82,21 +83,22 @@ ValueType accumulate(
 {
   typedef typename GlobInputIt::index_type index_t;
 
-  auto myid           = dash::myid();
+  auto & team         = in_first.team();
+  auto myid           = team.myid();
   auto index_range    = dash::local_range(in_first, in_last);
   auto l_first        = index_range.begin;
   auto l_last         = index_range.end;
   auto l_result = std::accumulate(l_first, l_last, init, binary_op);
 
-  dash::Array<index_t> l_results(dash::size());
+  dash::Array<index_t> l_results(team.size());
 
   l_results.local[0] = l_result;
 
   in_first.pattern().team().barrier();
```

Comment:
I assume `in_first.pattern().team().barrier()` is the same as `team.barrier()` here? A unified use would improve readability.